modules/simple-payments/simple-payments.php

```diff
@@ -197,31 +201,50 @@ function output_admin_warning( $data ) {
 		) );
 	}
 
-	function output_shortcode( $data ) {
+	/**
+	 * Get the HTML output to use as PayPal purchase box.
+	 *
+	 * @param string  $dom_id ID of the DOM element with the purchase message.
+	 * @param boolean $is_multiple Whether multiple items of the same product can be purchased.
+	 *
+	 * @return string
+	 */
+	public function output_purchase_box( $dom_id, $is_multiple ) {
```

Comment:
Do we need to add a default to `$is_multiple` here?

Reply:
Unless there is any styling guideline I'm unaware of that enforces to add default parameters, I think we're good here (this function is used currently twice, and both of them explicitly pass a value for `$is_multiple`).